client/src/components/JurisdictionAdmin/generateSheets.ts

```diff
@@ -150,63 +166,135 @@ export const downloadAuditBoardCredentials = async (
   jurisdictionName: string,
   auditName: string
 ): Promise<string> => {
-  const auditBoardsWithoutBallots: string[] = []
-  const auditBoardCreds = new jsPDF({ format: 'letter' })
-  auditBoards.forEach((board, i) => {
+  const doc = new jsPDF({ format: 'letter', unit: 'pt' })
+
+  const auditBoardsWithBallots = auditBoards.filter(
+    board => board.currentRoundStatus.numSampledBallots > 0
+  )
+  const auditBoardsWithoutBallots = auditBoards.filter(
+    board => board.currentRoundStatus.numSampledBallots === 0
+  )
+  auditBoardsWithBallots.forEach((board, i) => {
     const qr: HTMLCanvasElement | null = document.querySelector(
       `#qr-${board.passphrase} > canvas`
     )
     /* istanbul ignore next */
     if (!qr) return
-    if (board.currentRoundStatus.numSampledBallots > 0) {
-      if (i > 0) auditBoardCreds.addPage('letter')
-      const url = qr.toDataURL()
-      auditBoardCreds.setFont('Helvetica', '', 'bold')
-      auditBoardCreds.setFontSize(22)
-      auditBoardCreds.text(board.name, 20, 20)
-      auditBoardCreds.setFont('Helvetica', '', 'normal')
-      auditBoardCreds.setFontSize(14)
-      auditBoardCreds.text(
-        'Scan this QR code to enter the votes you see on your assigned ballots.',
-        20,
-        40
-      )
-      auditBoardCreds.addImage(url, 'JPEG', 20, 50, 50, 50)
-      auditBoardCreds.text(
-        auditBoardCreds.splitTextToSize(
-          'If you are not able to scan the QR code, you may also type the following URL into a web browser to access the data entry portal.',
-          180
-        ),
-        20,
-        120
-      )
-      const urlText: string[] = auditBoardCreds.splitTextToSize(
-        `${window.location.origin}/auditboard/${board.passphrase}`,
-        180
-      )
-      const urlHeight = urlText.reduce(
-        (a: number, t: string) => auditBoardCreds.getTextDimensions(t).h + a,
-        0
-      )
-      auditBoardCreds.text(urlText, 20, 140)
-      auditBoardCreds.link(0, 130, 220, urlHeight + 10, {
-        url: `${window.location.origin}/auditboard/${board.passphrase}`,
-      })
-    } else {
-      auditBoardsWithoutBallots.push(board.name)
-    }
+    if (i > 0) doc.addPage('letter')
+
+    let y = pageMargin
+    const x = pageMargin
+    const wrapWidth = pageContentWidth
+    doc.setFont('Helvetica', 'normal').setFontSize(defaultFontSize)
+
+    doc.setFillColor('#F8EDE2') // Light yellow from Blueprint
```

Comment:
Was gonna suggest importing this color directly from blueprint, but I couldn't seem to find it 🤷

```ts
import { Colors } from '@blueprintjs/core'
```

<img width="608" alt="colors" src="https://user-images.githubusercontent.com/12616928/189989574-f73f06e7-d8e0-421b-b808-4091392097df.png">

Reply:
Oh yeah it's a tint I got from the web inspector. But I'll try those golds and see if any work

Reply:
Yeah seems like they don't offer any light tints because they use opacity to lighten them in their CSS, which we can't do with jsPDF

Reply:
Got it! Thanks for looking into that